app/parse.py

```diff
@@ -15,8 +31,126 @@ class Product:
     num_of_reviews: int
 
 
+def init_driver() -> webdriver.Chrome:
+    options = Options()
+    options.add_argument("--headless")
+    options.add_argument("--no-sandbox")
+    options.add_argument("--disable-dev-shm-usage")
+
+    driver = webdriver.Chrome(
+        service=ChromeService(ChromeDriverManager().install()), options=options
+    )
+    return driver
+
+
+def accept_cookies(driver: webdriver.Chrome) -> None:
+    try:
+        accept_button = WebDriverWait(driver, 2).until(
+            ec.element_to_be_clickable((By.CLASS_NAME, "acceptCookies"))
+        )
+        accept_button.click()
+        print("Cookies accepted")
+    except Exception as e:
+        print(f"Cookies not accepted: {e}")
```

Comment:
The exception handling here is too broad. Catching a general `Exception` can make debugging difficult because it will catch all exceptions, including those you might not expect. Consider catching more specific exceptions, such as `TimeoutException` or `NoSuchElementException`, which are more likely to occur in this context.